### ISSUE TITLE
The dictionary/get_fields_data/<datasetid>/ API endpoint always returns NGT data

### ISSUE THREAD

**Issue by vanlummelhuizen** [Signbank/Global-signbank#1280]

**Comment by susanodd** [comment]
This probably happens if the user is not logged in?
The code (as written before adding the API token to it) had a fall-back to NGT when the user was not logged in, since it is public.

**Comment by vanlummelhuizen** [comment]
> This probably happens if the user is not logged in?

No, when API authentication is correct (the user is logged in), the result was still only for NGT. For instance, `curl --header 'Authorization:Bearer tSQICBglJoAYuGtz' https://signbank.cls.ru.nl/dictionary/get_fields_data/2/ ` (Kata Kolok) should give

```json
{
    "fields": [
        "Lemma ID Gloss: Balinees",
        "Lemma ID Gloss: Indonesisch",
        "Lemma ID Gloss: Engels",
        "Annotation ID Gloss: Balinees",
        "Annotation ID Gloss: Indonesisch",
        "Annotation ID Gloss: Engels",
        "Senses: Balinees",
        "Senses: Indonesisch",
        "Senses: Engels",
        ...
    ]
}
```

but gives

```json
{
  "fields": [
    "Lemma ID Gloss: Dutch",
    "Lemma ID Gloss: English",
    "Annotation ID Gloss: Dutch",
    "Annotation ID Gloss: English",
    "Senses: Dutch",
    "Senses: English",
    ...
  ]
}
```

This is now fixed in PR #1279 .

**Comment by vanlummelhuizen** [comment]
Since PR #1279 is merged, I am closing this issue.
